breeze

```diff
@@ -3536,13 +3505,6 @@ function breeze::run_breeze_command() {
         docker_engine_resources::check_all_resources
         runs::run_prepare_provider_documentation "${@}"
         ;;
-    perform_push_image)
```

Comment:
We do not need to push image from breeze any more. 'prepare cache' takes care about it via --cache-to directive during the build.